sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1452,6 +1512,12 @@ class LassoCV(RegressorMixin, LinearModelCV):
         (setting to 'random') often leads to significantly faster convergence
         especially when tol is higher than 1e-4.
 
+    cv_weighted : bool, default=False
+        If `True`, the MSE over folds is calculated as a weighted average,
+        weighted by sample_weight of the test fold. Here, sample_weight=None is
+        the same as `1` which means the weight is the number of observations in
+        the test folds.
```

Comment:
this last sentence is not clear enough I feel

Reply:
I tried to find a clearer formulation.